docs/space-amp.md

```diff
@@ -0,0 +1,57 @@
+## Space Amp
+
+Space amp is loosely defined the total storage space used to store a database on
+media, divided by total size of the user's data.  If a database has 1000 10-byte
+keys, each with 90 byte values, then size of the user's data is 100,000 bytes.
+If the files that store this data use 131,072 of storage, then the space amp is 1.31.
+
+### Sources of Space Amp
+
+There are two sources of space amp: overhead and garbage.
+
+#### Overhead Space Amp
```

Comment:
Overhead is a perfectly fine title since you are within the Sources of Space Amp header. Similar for the Garbage header

Reply:
Agree.  But I don't think the redundancy hurts.  And since overhead can mean many things, I prefer to leave it.